src/opensearchenlist/msdtc_enlist.cpp

```diff
@@ -1111,7 +1111,10 @@ RETCODE static EnlistInDtc_1pipe(void *conn, ITransaction *pTra,
                     case DTC_CHECK_RM_CONNECTION:
                         if (!confirmingLink) {
                             confirmingLink = true;
+
+                            // cppcheck-suppress unknownMacro
                             strcat(dtcname, ";" KEYWORD_DTC_CHECK "=0");
```

Comment:
As suggested. Would be nice to _actually fix this_, but (looking back at the Git history) looks like this error has been here since at least 2020, so probably not worth effort to try to track this down...